testing/helpers/fileManagerHelpers.js

```diff
@@ -219,7 +219,8 @@ export class FileManagerWrapper {
     }
 
     getToolbarElements() {
-        return this._$element.find(`.${Consts.TOOLBAR_CLASS} .${Consts.BUTTON_TEXT_CLASS}:visible, .${Consts.TOOLBAR_CLASS} .${Consts.NATIVE_TOOLBAR_CLASS}:visible .${Consts.DROP_DOWN_BUTTON_CLASS}`);
+        return this._$element.find(`.${Consts.TOOLBAR_CLASS} .${Consts.BUTTON_TEXT_CLASS}, .${Consts.TOOLBAR_CLASS} .${Consts.NATIVE_TOOLBAR_CLASS} .${Consts.DROP_DOWN_BUTTON_CLASS}`)
```

Comment:
This code checked some concrete parent elements' visibility, but now it gets all final elements and filters its by visibility. Is it expected?

Reply:
This may be confusing, but unfortunately we cannot use the `:visible` selector in the `document.querySelectorAll` method, so I moved this visibility check outside of the selector. It should work in the same manner.

Reply:
@EugeniyKiyashko  how do u think, can this change make test work another way?